adoc/helm_chart_override.adoc

```diff
@@ -12,14 +12,81 @@ You can adjust the {helm} chart installation in one of the following ways:
 [source]
 $ helm install <RELEASE_NAME> \
 oci://registry.suse.com/private-registry/private-registry-helm \
+--namespace <PRIVATE_REGISTRY_NAMESPACE> \
 --set harborAdminPassword=<MY_PASSWORD> \
---set externalURL=https://<MY_PRIVATE_REGISTRY_FQDN> \
---set expose.ingress.hosts.core=<MY_PRIVATE_REGISTRY_FQDN>
-* Create a custom `my-values.yaml` file and pass it to the `--values` flag, for example:
+--set externalURL=https://<PRIVATE_REGISTRY_FQDN> \
+--set expose.ingress.hosts.core=<PRIVATE_REGISTRY_FQDN>
+* Create a {suse} custom `suse_registry_override.yaml` file and pass it to the `--f` flag, for example:
 [source]
-$ helm install <RELEASE_NAME> \
-oci://registry.suse.com/private-registry/private-registry-helm \
---values my-values.yaml
+  $ helm install <RELEASE_NAME> \
+  oci://registry.suse.com/private-registry/private-registry-helm \
+  --namespace <PRIVATE_REGISTRY_NAMESPACE>
+  -f suse_registry_override.yaml
+
+[#pr-helm-chart-override-examples]
+=== Examples of {sregistry} {helm} override files
+.Minimal deployment with {ingress}
+[example]
+----
+expose:   
+  type: ingress //<1>
+  ingress:     
+    hosts:
+      core: <PRIVATE_REGISTRY_FQDN> //<2>
+
+externalURL: https://<PRIVATE_REGISTRY_FQDN> //<3>
+
+harborAdminPassword: "<MY_PASSWORD>" //<4>
+
+database:   
+  internal:     
+    password: "<MY_PASSWORD_POSTGRESQL>"
+
+redis:   
+  internal:     
+   password: "<MY_PASSWORD_REDIS>"
+----
+[callouts]
+<1> How {sregistry} is exposed. Can be `ingress`, `loadBalancer`, `nodePort` or `clusterIPhis`. Default is `ingress`.
```

Comment:
@Mauraza Do we / can we assume that a k8s operator will know the difference between these settings?

Reply:
Yes, they are the different types of [services in k8s](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) and [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)